src/MusicStore/Get-Crossgen.ps1

```diff
@@ -32,9 +32,9 @@ function Get-CoreCLRVersion()
 
     foreach ($name in $json["libraries"].Keys)
```

Comment:
There are 2 places today that I know of that uses crossgen:

1. The new `dotnet cache` command.  It finds the `clrjit` assembly using MSBuild, and then does a relative path to the `tools` directory: https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets#L66-L67.  I don't think that really helps you here, since you aren't inside MSBuild.
2. The CLI build infrastructure.  It builds up a path using the NuGet package directory as the root:  
https://github.com/dotnet/cli/blob/master/build/CrossGen.props#L4-L5.  If you know the root NuGet packages directory, you could do the same.

Or, if you want to continue calling `nuget install` but use `runtime.$(CoreCLRRid).microsoft.netcore.app` for the package name, you could do that too.  That way you can use the same version as you use for `Microsoft.NETCore.App`.